mlrun/common/schemas/api_gateway.py

```diff
@@ -103,11 +103,11 @@ def get_function_names(self):
         ]
 
     def get_invoke_url(self):
-        return (
-            self.spec.host + self.spec.path
-            if self.spec.path and self.spec.host
-            else self.spec.host
-        ).rstrip("/")
+        if self.spec.host and self.spec.path:
+            if not self.spec.path.startswith("/"):
+                return f"{self.spec.host.rstrip('/')}/{self.spec.path}"
+            return f"{self.spec.host.rstrip('/')}{self.spec.path}"
```

Comment:
You can `rstrip` the host and `lstrip` the path in both cases, without an if condition

Reply:
Also, we previously removed the last slash to not duplicate urls with and without slash suffix.
Make sure this is still the case (should be done in the BE IIRC)